open_dread_rando/tilegroup_patcher.py

```diff
@@ -0,0 +1,20 @@
+from open_dread_rando.patcher_editor import PatcherEditor
+
+def is_tilegroup(actor):
+    return "TILEGROUP" in actor.pComponents and "CBreakableTileGroupComponent" == actor.pComponents.TILEGROUP["@type"]
```

Comment:
tiny thing, but it's a bit easier to read if you do `actor.pComponents.TILEGROUP["@type"] == "CBreakableTileGroupComponent"` instead of the reverse as you have it rn

Reply:
Fixed in 9d727d4